### PR TITLE
Crashlytics Update CHANGELOG.md for v4.0.0-beta.5

### PR DESCRIPTION
**Pull request by samedson**

### CODE REVIEW

File: Crashlytics/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 
+# v4.0.0-beta.5
+
+- [changed] Updates two endpoints in the SDK with no expected end-user impact (#4953, #4988)
```
Comment (RelaxedPear) [inline Crashlytics/CHANGELOG.md]: "Changed"

Comment (RelaxedPear) [inline Crashlytics/CHANGELOG.md]: "Firebase Crashlytics SDK"

Comment (RelaxedPear) [inline Crashlytics/CHANGELOG.md]: Please add a period at the end (e.g., **(#4988).**), thanks :)

Comment (morganchen12) [reply Crashlytics/CHANGELOG.md]: These tags are consumed by a script and are supposed to be lowercase. They'll be imported to devsite as `* {{ changed }} Updates two endpoints ...`

Comment (morganchen12) [reply Crashlytics/CHANGELOG.md]: Oops, looks like I misread your comment. 👍

Comment (samedson) [reply Crashlytics/CHANGELOG.md]: I think it was in reference to "Updates two..." => "Changed two..."

**Pull request merged**